<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Feedback</class>
 <widget class="QDialog" name="Feedback">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>795</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnsubmit">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>480</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>671</width>
     <height>361</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
    border: 1px solid #cccccc;
    border-radius: 5px;
    margin-top: 10px;
    font: bold 20px Arial;
    color: #666;
    padding: 10px;
}

QGroupBox:title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 0 3px;
}
</string>
   </property>
   <property name="title">
    <string>&quot;GIVE FEEDBACK&quot;</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter Vehicle ID:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineVehicleId">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>461</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Rate your experience(1-5):</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboRating">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>140</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
    background-color: #f0f0f0;
    color: #777777;
    font: 12px Arial, sans-serif;
    padding: 4px 6px;
    border: 1px solid #cccccc;
    border-radius: 3px;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 20px;
    border-left: 1px solid #ccc;
}

QComboBox QAbstractItemView {
    background-color: #ffffff;
    color: #555555;
    selection-background-color: #a0a0a0;
    selection-color: white;
}
</string>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QTextEdit" name="textFeedback">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>210</y>
      <width>451</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit {
    background-color: #fdfdfd;
    color: #777777;
    font: 12px Arial, sans-serif;
    border: 1px solid #cccccc;
    border-radius: 4px;
    padding: 6px;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Feedback:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
